components/common/EmptyView.tsx

```diff
@@ -0,0 +1,5 @@
+/**
+ * A view which renders nothing. This is mostly useful for default props
+ * on other components which may optionally show a view.
+ */
+export const EmptyView = () => <></>;
```

Comment:
We can also use {null} when we want to render a blank

Reply:
If that's more concise then I'm fine with doing that, it's just that in the SwiftUI world (and some other swift libraries), this `EmptyView` notion seems to be preferred because it's more explicit.